source/features/minimize-user-comments.tsx

```diff
@@ -88,7 +90,7 @@ async function minimizeMutedUserComments(): Promise<void> {
 function init(): void {
 	minimizeMutedUserComments();
 	onNewComments(minimizeMutedUserComments);
-	delegate('.timeline-comment-action', 'click', handleMenuOpening);
+	delegate('summary.timeline-comment-action:not([aria-label="Add your reaction"])', 'click', handleMenuOpening);
```

Comment:
`summary` isn't required here, it's best to leave it out

Reply:
Without it, edit buttons get selected too. Is that okay?